cmd/identity-api/main.go

```diff
@@ -51,9 +52,14 @@ func main() {
 
 	repo := repositories.New(dbs)
 
-	s := handler.NewDefaultServer(graph.NewExecutableSchema(graph.Config{Resolvers: &graph.Resolver{
+	cfg := graph.Config{Resolvers: &graph.Resolver{
 		Repo: repo,
-	}}))
+	}}
+	cfg.Directives.OneOf = func(ctx context.Context, obj interface{}, next graphql.Resolver) (res interface{}, err error) {
```

Comment:
```suggestion
	cfg.Directives.OneOf = func(ctx context.Context, _ any, _ graphql.Resolver) (any, error) {
```
Should we also leave a comment like "The directive on its own is advisory; everything is enforced inside of the resolver".